source/configuring_domain/05_configuring_domain/regional_settings.rst

```diff
@@ -18,3 +18,19 @@ To change the timezone of a domain, go to Configuration > Regional settings. In
    :class: note
 
    It is also possible to directly write a city or zone in the textbox. A selection of matching existing timezones will appear.
+
+
+Datasets default location
+-------------------------
+
+The :ref:`geographic coverage metadata <geographic-coverage-metadata>` indicates the location of the data contained in the dataset. Although the metadata can be set individually for each dataset, it is possible to configure the default location(s) that will be automatically set as geographic coverage metadata for all datasets created afterward.
```

Comment:
Is it possible to override the default location (when set) at the dataset level afterward?

Reply:
Yes, that's the "Although the metadata can be set individually for each dataset" part. IMO that was clear when reading, but I'm obviously biased 😬

Reply:
```suggestion
The :ref:`geographic coverage metadata <geographic-coverage-metadata>` indicates the location of the data contained in the dataset. Although the metadata can be set individually for each dataset afterward, it is possible to configure the default location(s) that will be automatically set as geographic coverage metadata for all datasets created afterward.
```